app/views/components/warning.scala.html

```diff
@@ -14,12 +14,14 @@
  * limitations under the License.
  *@
 
-@(message: String, containerClass: String = "")
+@import uk.gov.hmrc.govukfrontend.views.Aliases.{Text, WarningText}
+@import uk.gov.hmrc.govukfrontend.views.html.components.GovukWarningText
```

Comment:
Didn't add to `TwirlKeys.templateImports` as it is just one file and  there is a lot of hardcoded html i.e. not using the components from the library. Maybe in future when we use components from the play frontend hmrc library,  we can move to the build.sbt.